internal/api/handlers/requestadapters/schedulers.go

```diff
@@ -398,6 +482,41 @@ func getSpec(spec game_room.Spec) *api.Spec {
 	return nil
 }
 
+func getAutoscaling(autoscaling *autoscaling.Autoscaling) *api.Autoscaling {
+	if autoscaling != nil {
+		return &api.Autoscaling{
+			Enabled: autoscaling.Enabled,
+			Min:     int32(autoscaling.Min),
+			Max:     int32(autoscaling.Max),
+			Policy:  getAutoscalingPolicy(autoscaling.Policy),
+		}
+	}
+
+	return nil
+}
+
+func getAutoscalingPolicy(autoscalingPolicy autoscaling.Policy) *api.AutoscalingPolicy {
+	return &api.AutoscalingPolicy{
+		Type:       string(autoscalingPolicy.Type),
+		Parameters: getPolicyParameters(autoscalingPolicy.Parameters),
+	}
```

Comment:
```suggestion
        if autoscalingPolicy == nil {
            return nil
        }
        
	return &api.AutoscalingPolicy{
		Type:       string(autoscalingPolicy.Type),
		Parameters: getPolicyParameters(autoscalingPolicy.Parameters),
	}
```

Reply:
Actually this is impossible because autoscalingPolicy is not a pointer